src/puppetlabs/puppetdb/pdb_routing.clj

```diff
@@ -93,6 +94,16 @@
                    true
                    @maint-mode-atom))))
 
+(defn get-artifact-version
```

Comment:
I'd make a note in a docstring about how this needs to be fixed upstream in `clj-semver`